### PR TITLE
Fvk gft natural resources

### PR DESCRIPTION
**Pull request by fvankrieken**
First half of #672

### CODE REVIEW

File: dcpy/connectors/esri/arcgis_feature_service.py [hunk 1]
```diff
@@ -57,7 +57,7 @@ def query_dataset(dataset: FeatureServer, params: dict) -> dict:
 
 
 def get_dataset(dataset: FeatureServer, crs: int) -> dict:
-    CHUNK_SIZE = 2000
+    CHUNK_SIZE = 100
```
Comment (fvankrieken) [inline dcpy/connectors/esri/arcgis_feature_service.py:60]: @alexrichey any thoughts on if maybe this should just be an override for specific problem datasets? Gut is yes

Comment (fvankrieken) [reply dcpy/connectors/esri/arcgis_feature_service.py:60]: But also simple to just keep it low since performance isn't really an issue

Comment (alexrichey) [reply dcpy/connectors/esri/arcgis_feature_service.py:60]: Interesting... so does this resolve the issue with the sporadic failures? If so, I'd probably just go the simplest route, and just hardcode like you've done here.

Comment (fvankrieken) [reply dcpy/connectors/esri/arcgis_feature_service.py:60]: It did. NYS GIS Clearinghouse seemed to error out on any given query at exactly 1 minute

File: dcpy/library/templates/dcp_beaches.yml [hunk 2]
```diff
@@ -0,0 +1,23 @@
+dataset:
+  name: dcp_beaches
+  acl: public-read
+  source:
+    url:
+      path: s3://edm-recipes/inbox/dcp_beaches/nyc_beaches.zip
```
Comment (alexrichey) [inline dcpy/library/templates/dcp_beaches.yml:6]: grim thought: I hope we never need to introduce versions for this dataset

Comment (fvankrieken) [reply dcpy/library/templates/dcp_beaches.yml:6]: 🙁

File: dcpy/library/templates/nysdec_freshwater_wetlands_checkzones.yml [hunk 3]
```diff
@@ -0,0 +1,25 @@
+dataset:
+  name: nysdec_freshwater_wetlands
+  acl: public-read
+  source:
+    arcgis_feature_server:
+      server: nys_clearinghouse
+      name: State_Regulated_Freshwater_Wetlands
+      layer: 1
+    geometry:
+      SRS: EPSG:2263
```
Comment (alexrichey) [inline dcpy/library/templates/nysdec_freshwater_wetlands_checkzones.yml:10]: [This](https://services6.arcgis.com/DZHaqZm9cxOD4CWM/ArcGIS/rest/services/State_Regulated_Freshwater_Wetlands/FeatureServer/1) says `Spatial Reference: 26918 (26918)`. Does that matter?

Comment (fvankrieken) [reply dcpy/library/templates/nysdec_freshwater_wetlands_checkzones.yml:10]: I believe that's meant for the extent. Currently, this geometry gets fed into the call to arcgis connector, and queries for the data specifically in the given SRS

Comment (fvankrieken) [reply dcpy/library/templates/nysdec_freshwater_wetlands_checkzones.yml:10]: The REST service supports that. So to answer your question - not entirely, though it's maybe not great that there's this uncertainty about where transformation happens

File: dcpy/connectors/esri/arcgis_feature_service.py (hunk 1 above)
Comment (alexrichey) [reply dcpy/connectors/esri/arcgis_feature_service.py:60]: ah... interesting. Might make sense down the road to add some retry/backoff logic, but this seems great for now

**Review by alexrichey — APPROVED** [review]

**Pull request merged**